src/targets/linux-uhid/main.c

```diff
@@ -159,10 +117,10 @@ int main(void)
 	strcpy(create.name, config.device_name);
 	memcpy(create.rd_data, oi_rdesc, sizeof(oi_rdesc)); /* protocol report descriptor */
 	rdesc_size += sizeof(oi_rdesc);
-	memcpy(create.rd_data + rdesc_size, rdesc, sizeof(rdesc)); /* mouse report descriptor */
```

Comment:
this now fails the code format check, because its longer, use the following:
```
-	memcpy(create.rd_data + rdesc_size, desc_hid_mouse_report, sizeof(desc_hid_mouse_report)); /* mouse report descriptor */

+	memcpy(create.rd_data + rdesc_size,
+	       desc_hid_mouse_report,
+	       sizeof(desc_hid_mouse_report)); /* mouse report descriptor */
```